aws/logs_monitoring/cache.py

```diff
@@ -442,3 +442,36 @@ def get(self, log_group):
             self.tags_by_id[log_group] = log_group_tags
 
         return log_group_tags
+
+
+resource_group_tagging_client = boto3.client("resourcegroupstaggingapi")
+RESOURCES_PER_PAGE = 100
+LRU_CACHE_SIZE = 50
+STEP_FUNCTIONS_TAGS_CACHE_TTL = 3600  # an hour
+
+
+def get_ttl_hash(ttl=STEP_FUNCTIONS_TAGS_CACHE_TTL) -> float:
+    """Return the same value within `ttl` seconds"""
+    return time() // ttl
+
+
+@lru_cache(maxsize=LRU_CACHE_SIZE)
+def get_step_function_tags(
+    step_function_arn: str, time_hash: float = get_ttl_hash()
```

Comment:
I don't think time_hash is being used here.

Reply:
Yes, going to remove the `@lru_cache` and this `time_hash` since we're going to implement something similar to `CloudwatchLogGroupTagsCache` or `LambdaTagsCache`.